### PR TITLE
feat(FWS-85): Localization

### PR DESCRIPTION
**Pull request by emilegirardGit**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-85

Context:
This ticket was about making the website in English and in french.

Changes
-Added button in the footer (English, Francais)
-Added i18n libraries for static string need translation
-Added english en french fields in the database

UI Changes
localization

### CODE REVIEW

**Review by Eli293 — APPROVED** [review]
Looks good.

File: fitwsarah_frontend/src/components/authentication/RoleBasedSwitch.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { isRole } from './authUtils';
 import "../authentication/switch.css"
+import { useTranslation } from "react-i18next";
```
Comment (AlexeiTimbro) [inline fitwsarah_frontend/src/components/authentication/RoleBasedSwitch.js:4]: Do we have to use this import in every one of our file to allow the localization to work?

File: fitwsarah_frontend/src/LanguageConfig/i18n.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+.use(initReactI18next)
+.use(Backend)
+.init({
+    debug: false,
+    fallbackLng: 'en',
+    interpolation: {
+        escapeValue: false,
+    },
+    backend: {
+        loadPath: '/LanguageContext/{{lng}}/{{ns}}.json',
```
Comment (AlexeiTimbro) [inline fitwsarah_frontend/src/LanguageConfig/i18n.js:17]: Could you elaborate on how loading this .json file makes the localization work?

**Review by AlexeiTimbro — APPROVED** [review]
Great work I left comment resolve conversation before merging down below

File: fitwsarah_frontend/src/components/authentication/RoleBasedSwitch.js (hunk 1 above)
Comment (emilegirardGit) [reply fitwsarah_frontend/src/components/authentication/RoleBasedSwitch.js:4]: Yes you need to import in every files

File: fitwsarah_frontend/src/LanguageConfig/i18n.js (hunk 2 above)
Comment (emilegirardGit) [reply fitwsarah_frontend/src/LanguageConfig/i18n.js:17]: I will explain in a call its easy but will be easier if I show you

**Pull request merged**